webhook_to_fedora_messaging_messages/github/__init__.py

```diff
@@ -0,0 +1,132 @@
+
+def summarize_repository_event(event_type: str, payload: dict) -> str:
+    
+    if event_type in ['push', 'fork']:
+        summary = _handle_repository_event(event_type, payload)
+    elif event_type.startswith('pull_request'):
+        summary = _handle_pull_request_event(event_type, payload)
+    elif event_type.startswith('issue'):
+        summary = _handle_issue_event(event_type, payload)
+    else:
+        summary = "Event type not supported"
+
+    return summary
+
+
+def _handle_repository_event(event_type: str, payload: dict) -> str:
+    if event_type == 'push':
+        return _summarize_push_event(payload)
+    elif event_type == 'fork':
+        return _summarize_fork_event(payload)
+    else:
+        raise NotImplementedError
+    
+def _handle_pull_request_event(event_type: str, payload: dict) -> str:
```

Comment:
I was going to say the same thing about the prefix but in this case you can just drop this function :-)